// this program contains a bug. the Println error is unhandled.

package main

import "fmt"

func main() {
	fmt.Println("Hello world")
}
